docs/source/usage.rst

```diff
@@ -757,6 +757,18 @@ as DictConfig, allowing attribute style access on the resulting node.
     >>> assert type(container["structured_config"]) is DictConfig
     >>> assert container["structured_config"].port == 80
 
+Using **structured_config_mode=SCMode.INSTANTIATE** causes such nodes to
+be converted to instances of the backing dataclass.
```

Comment:
1. document to_object instead and mention that it's calling to_container in with SCMode.INTANTIATE?
2. backing dataclass is inaccurate (attr classes too, maybe other things later). Underlying Structured Config type?
3. This could be too long to fit nicely in the code block, check it in a browser.

Reply:
Ok, I made some updates in https://github.com/omry/omegaconf/pull/502/commits/32f6c68c6dde51365bc15d94fa99e54388bbfa41.

Reply:
>     3. This could be too long to fit nicely in the code block, check it in a browser.

You were right, it's fixed now.

I also noticed that the `OmegaConf.to_container` section has a very long line that does not fit in the code block. I opened up PR https://github.com/omry/omegaconf/pull/614 to address it.